docs/apis-tools/camunda-api-rest/guidelines/data-fetching.md

```diff
@@ -0,0 +1,324 @@
+---
+id: camunda-api-rest-guidelines-data
+title: "Data fetching"
+description: "Learn about the basic guidelines, structures, and conventions of the Camunda 8 REST API."
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+All persistent resources like process definitions, user tasks, users, or tenants provide at least one endpoint to fetch related data. Most of those endpoints provide data with **near-real time consistency** queried from exported records, if records for the respective resource are exported by the engine. If resources are not based on exported records, such as license data or topology information, the data returned by those endpoints can reflect real time insights or static content. For resources like messages, signals, or the engine's internal clock, the engine does not persist any specific data and thus there is no API to fetch them.
```

Comment:
Is there somewhere else I can learn about persistent resources? Do we cover this anywhere else in the docs?